kaizen/cli/commands/memory_inspector.py

```
@@ -0,0 +1,401 @@
+    # Detailed inspection
+    inspector.show_detailed_structure()
+    
+    # Export to file
+    inspector.export_to_file("memory_dump.json")
+    
+    # Get specific data
+    test_runs = inspector.get_test_runs()
+    fix_attempts = inspector.get_fix_attempts()
+"""
+
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)